data/data/rhcos.json

```diff
@@ -1,104 +1,111 @@
 {
     "amis": {
         "ap-northeast-1": {
-            "hvm": "ami-0db1f56b7e43c2dcf"
+            "hvm": "ami-0c63b39219b8123e5"
         },
         "ap-northeast-2": {
-            "hvm": "ami-037ba98a635353cd5"
+            "hvm": "ami-073cba0913d2250a4"
         },
         "ap-south-1": {
-            "hvm": "ami-06ca6319f9a7cc7ef"
+            "hvm": "ami-0270be11430101040"
         },
         "ap-southeast-1": {
-            "hvm": "ami-09dc67bc03673a533"
+            "hvm": "ami-06eb9d35ede4f08a3"
         },
         "ap-southeast-2": {
-            "hvm": "ami-0005fdca8c2b639cd"
+            "hvm": "ami-0d980796ce258b5d5"
         },
         "ca-central-1": {
-            "hvm": "ami-0937ba51d6915aae8"
+            "hvm": "ami-0f907257d1686e3f7"
         },
         "eu-central-1": {
-            "hvm": "ami-0060b31e9ef744e3d"
+            "hvm": "ami-02fdd627029c0055b"
         },
         "eu-west-1": {
-            "hvm": "ami-008aa9cb0e9723ae7"
+            "hvm": "ami-0d4839574724ed3fa"
         },
         "eu-west-2": {
-            "hvm": "ami-02ff9f8182d312adc"
+            "hvm": "ami-053073b95aa285347"
         },
         "eu-west-3": {
-            "hvm": "ami-064c1a19b5600d4bb"
+            "hvm": "ami-09deb5deb6567bcd5"
         },
         "sa-east-1": {
-            "hvm": "ami-0ee0e369b4244858d"
+            "hvm": "ami-068a2000546e1889d"
         },
         "us-east-1": {
-            "hvm": "ami-033b8fc59b43c23ee"
+            "hvm": "ami-046fe691f52a953f9"
         },
         "us-east-2": {
-            "hvm": "ami-02200f690a88f0819"
+            "hvm": "ami-0649fd5d42859bdfc"
         },
         "us-west-1": {
-            "hvm": "ami-03fc8933dc66db8f0"
+            "hvm": "ami-0c1d2b5606111ac8c"
         },
         "us-west-2": {
-            "hvm": "ami-096cb19297a5c2726"
+            "hvm": "ami-00745fcbb14a863ed"
         }
     },
-    "baseURI": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/ootpa/410.8.20190508.1/",
-    "buildid": "410.8.20190508.1",
+    "baseURI": "https://releases-art-jenkins.cloud.paas.upshift.redhat.com/storage/releases/ootpa/410.8.20190520.0/",
```

Comment:
The original URL is the public redirected one. Although if this is the GA candidate, do we want a more "official" location? If not the original baseURI doesn't need to be changed

Reply:
No more `baseURI` with 9f1e50d.